modules/custom-tuning-example.adoc

```diff
@@ -43,3 +42,45 @@ Custom profile writers are strongly encouraged to include the default TuneD
 daemon profiles shipped within the default Tuned CR. The example above uses the
 default `openshift-control-plane` profile to accomplish this.
 ====
+
+*Using built-in TuneD profiles*
+
+Given the successful rollout of the NTO-managed DaemonSet,
+the TuneD operands all manage the same version of the
+link:https://github.com/redhat-performance/tuned/[TuneD daemon].  To list
```

Comment:
We can't link to GitHub pages unless we get special permission from Support
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites